src/kudu/consensus/consensus_peers.cc

```diff
@@ -128,6 +132,7 @@ Peer::Peer(RaftPeerPB peer_pb,
            string tablet_id,
            string leader_uuid,
            PeerMessageQueue* queue,
+           PeerProxyPool* peer_proxy_pool,
            ThreadPoolToken* raft_pool_token,
            shared_ptr<PeerProxy> proxy,
```

Comment:
there is overlap of terminology between Proxy in PeerProxy and proxy in PeerProxyPool.

Reply:
This is correct. It's unfortunate but I don't think it's a good idea to go in and get rid of the existing usage of the Proxy term since it would be a huge diff. Do you have any suggestions?

Reply:
Not suggesting change PeerProxy, but new code could use different terminology.